### PR TITLE
update(docs): update repo/website cross referencing for rules

### PR DESCRIPTION
**Pull request by LucaGuerra**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind content

/area documentation

**What this PR does / why we need it**:

This PR adds cross-referencing to the rules repository from the appropriate documentation pages.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/docs/rules/_index.md [hunk 1]
```diff
@@ -19,3 +19,5 @@ Element | Description
 :-------|:-----------
 `required_engine_version` | Used to track compatibility between rules content and the falco [engine version](/docs/rules/versioning/#falco-engine-versioning).
 `required_plugin_versions` | Used to track compatibility between rules content and [plugin versions](/docs/plugins#plugin-versions-and-falco-rules).
+
+The Falco organization maintains a [rules repository](https://github.com/falcosecurity/rules) that provides both easy to install rules and examples for rule writers. You can learn more about the [default and custom rulesets](/docs/rules/default-custom) in the documentation.
```
Comment (nate-double-u) [inline content/en/docs/rules/_index.md:23]:
```suggestion
The Falco organization maintains a [rules repository](https://github.com/falcosecurity/rules) that provides easy-to-install rules and examples for rule writers. You can learn more about the [default and custom rulesets](/docs/rules/default-custom) in the documentation.
```

File: content/en/docs/rules/versioning.md [hunk 2]
```diff
@@ -20,7 +16,7 @@ Element | Description
 
 ### Falco Engine Versioning
 
-The `falco` executable and the `falco_engine` C++ object now support returning a version number. The initial version is 2 (implying that prior versions were 1). We will increment this version any time we make an incompatible change to the rules file format or add new filtercheck fields/operators to Falco.
+The `falco` executable and the `falco_engine` C++ object now support returning a version number. The initial version is 2 (implying that prior versions were 1). We will increment this version any time we make an incompatible change to the rules file format or add new filtercheck fields/operators to Falco. You can check the Falco engine version that your installation supports by running `falco --version`.
```
Comment (nate-double-u) [inline content/en/docs/rules/versioning.md:19]:
```suggestion
The `falco` executable and the `falco_engine` C++ object now supports returning a version number. The initial version is 2 (implying that prior versions were 1). We will increment this version whenever we make an incompatible change to the rules file format or add new filtercheck fields/operators to Falco. You can check the Falco engine version that your installation supports by running `falco --version`.
```
Should "filtercheck" be in a code fence like the command? (is it, or does it need to be in the glossary?)

**Review by nate-double-u — COMMENTED** [review]
A couple comments inline:

File: content/en/docs/rules/versioning.md (hunk 2 above)
Comment (LucaGuerra) [reply content/en/docs/rules/versioning.md:19]: Great point, I'll update and remove the "filtercheck" wording from here because it's not the right place.

Comment (LucaGuerra) [reply content/en/docs/rules/versioning.md]: Done!

**Review by leogr — APPROVED** [review]

**Pull request merged**